### PR TITLE
Fix missing renames for item->at

### PR DESCRIPTION
**Pull request by leobalter**
includes nit pick using verifyProperty

Ref #2905

### CODE REVIEW

**Review by rwaldron — DISMISSED** [review]

**Comment by rkirsling** [comment]
Thanks @leobalter! :bow:

File: test/built-ins/TypedArray/prototype/at/prop-desc.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ assert.sameValue(
   'The value of `typeof TypedArray.prototype.at` is "function"'
 );
 
-verifyNotEnumerable(TypedArray.prototype, 'item');
-verifyWritable(TypedArray.prototype, 'item');
-verifyConfigurable(TypedArray.prototype, 'item');
+verifyProperty(TypedArray.prototype.at, 'name', {
```
Comment (rkirsling) [inline test/built-ins/TypedArray/prototype/at/prop-desc.js:20]: Oops, this and the String one are mistakenly checking `name` here. Will submit another PR to fix.
